packages/usa-date-range-picker/src/usa-date-range-picker.json

```diff
@@ -0,0 +1,7 @@
+{
+  "defaultValueStart": "",
+  "defaultValueEnd": "",
```

Comment:
@mejiaj I have updated the controls used to be `text` instead of `date`, since that is what the attribute was really looking for. I also updated the variable and control names to be a bit more intuitive. Please let me know if you have feedback!